text/0698-deprecate-link-to-positional-arguments.md

```diff
@@ -0,0 +1,178 @@
+---
+Stage: Accepted
+Start Date: 2021-01-05
+Release Date: Unreleased
+Release Versions:
+  ember-source: vX.Y.Z
+  ember-data: vX.Y.Z
+Relevant Team(s): Ember.js
+RFC PR: https://github.com/emberjs/rfcs/pull/698
+---
+
+<!---
+Directions for above:
+
+Stage: Leave as is
+Start Date: Fill in with today's date, YYYY-MM-DD
+Release Date: Leave as is
+Release Versions: Leave as is
+Relevant Team(s): Fill this in with the [team(s)](README.md#relevant-teams) to which this RFC applies
+RFC PR: Fill this in with the URL for the Proposal RFC PR
+-->
+
+# Deprecate <LinkTo> Component Positional Arguments
+
+## Summary
+
+We propose to deprecate invoking the `<LinkTo>` component with positional
+arguments, in favor of the equivilant named arguments introduced in
+[RFC #459](0459-angle-bracket-built-in-components.md). We also propose to
+deprecate the `(query-params)` helper, which is only needed when invoking the
+`<LinkTo>` component with positional arguments.
+
+## Motivation
+
+In modern Ember, the idomatic way to invoke most componnts is to use the
+[angle bracket syntax](0311-angle-bracket-invocation.md) along with the named
+arguments `@` syntax. On the other hand, curly invocations are now reserved for
+"helper-like" and "control-flow" components.
+
+The `<LinkTo>` built-in component started life as a "helper" in the early days
+of Ember, even before an official components API was created. Over time, it
+became clear that `<LinkTo>` is better classified as a component than a helper
+in modern Ember, and the learning materials have been updated accordingly.
```

Comment:
https://guides.emberjs.com/release/routing/linking-between-routes/#toc_the-linkto--component

Do we want to reference any links for future readers or are they at risk to change?

Reply:
Thanks for including the link here so people reviewing the PR can find them. That said, there are also other places, such as [the tutorial](https://guides.emberjs.com/release/tutorial/part-1/building-pages/). This is meant to include all of these so I don't think we need to link from the RFC text itself.